src/openforms/authentication/contrib/digid/views.py

```diff
@@ -43,7 +43,12 @@ def get_level_of_assurance(self):
         _, _, kwargs = resolve(return_path)
         form = get_object_or_404(Form, slug=kwargs.get("slug"))
 
-        loa = form.authentication_backend_options.get(PLUGIN_ID, {}).get("loa")
+        loa = None
+        plugin_configuration = form.authentication_backends_configuration.filter(
+            backend=PLUGIN_ID
+        ).first()
```

Comment:
I'd use a `.get(backend=PLUGIN_ID)` instead to hint that at most one record is expected (and then you'll need a try catch of course)